cmd/git-sync/webhooks.go

```diff
@@ -0,0 +1,71 @@
+
+package main
+
+import (
+	"net/http"
+	"time"
+	"fmt"
+)
+// Create an http client that has our timeout by default
+var netClient = &http.Client{
+	Timeout: time.Duration(time.Second * time.Duration(*flWebhookTimeout) ),
+}
+
+// Trigger channel for webhook requests. If anything is received into this channel
+//   it triggers the webhook goroutine to send new requests.
+var WebhookCallTriggerChannel = make(chan struct{})
```

Comment:
Rather than make this a global, why not create it in `main` and pass it into `ServeWebhooks()` as an argument